server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/CBApplication.java

```diff
@@ -338,10 +338,7 @@ protected void startServer() {
             // Try to configure automatically
             performAutoConfiguration(configPath.toFile().getParentFile());
         } else if (appConfiguration.isGrantConnectionsAccessToAnonymousTeam()) {
-            String autoAdminName = System.getenv(CBConstants.VAR_AUTO_CB_ADMIN_NAME);
-            if (autoAdminName != null) {
-                grantAnonymousAccessToConnections(appConfiguration, autoAdminName);
-            }
+            grantAnonymousAccessToConnections(appConfiguration, CBConstants.DEFAULT_ADMIN_NAME);
```

Comment:
use 'auto-grant' instead of DEFAULT_ADMIN_NAME, since such an administrator may not exist